components/Tabs/Tabs.js

```diff
@@ -64,9 +90,15 @@ class TabCard {
 
 - Select all classes named ".tab" and assign that value to the tabs variable
 
+
 - With your selection in place, now chain a .forEach() method onto the tabs variable to iterate over the DOM NodeList
 
-- In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
+- In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter */
+
+
+var tabs = document.querySelectorAll(".tab");
+
+tabs.forEach(function(tabElement){
+  return new TabLink(tabElement);
+})
```

Comment:
This way is perfectly fine, but here is a way to do it in one line:

let tabs = document.querySelectorAll('.tab').forEach(tab => new TabLink(tab))